test/spec/collection_view_spec.coffee

```diff
@@ -412,51 +412,36 @@ define [
 
       it 'should animate the opacity of new items', ->
         return unless jQuery
-        $css = sinon.stub jQuery.prototype, 'css', -> this
         $animate = sinon.stub jQuery.prototype, 'animate', -> this
 
         createCollection()
         collectionView = new AnimatingCollectionView {collection}
 
-        expect($css.callCount).to.be collection.length
-        expect($css).was.calledWith 'opacity', 0
-
         expect($animate.callCount).to.be collection.length
```

Comment:
Wastn't so sure about all these tests. It felt weird to just test that style.opacity was '0' on all childs views.